packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,292 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconUnverified from 'terra-icon/lib/icon/IconDiamond';
+import Observation from './common/observation/_Observation';
+import observationPropShape from './proptypes/observationPropTypes';
+import ResultError from './common/other/_ResultError';
+import NoData from './common/other/_KnownNoData';
+import { isEmpty, checkIsStatusInError, ConditionalWrapper } from './common/utils';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Result Object with the clinical result data.
+   */
+  resultData: PropTypes.shape({
```

Comment:
Can we flatten API to:
```
  resultId: PropTypes.string, // or should this be eventID
  diastolicResultData: PropTypes.shape({
    result,
    status,
    conceptDisplay,
    datetimeDisplay,
    isModified,
    hasComment,
    isUnverified,
  }),
  systolicResultData: PropTypes.shape({
     result,
    status,
    conceptDisplay,
    datetimeDisplay,
    isModified,
    hasComment,
    isUnverified,
  }),
```
Also, looking through all the login in here, why does the systolic data props get to override the diastolic data when comparing the icons that are generated? Should these icons be tied to each concept?

IMO reusing these proptypes are introducing a some unused API and it making it hard to track down what is being used here verse else where.

Reply:
Each systolic and diastolic result is it's own distinct clinical result, so the api is an array of the two results and matches a FHIR [bloodpressure](https://www.hl7.org/fhir/observation-example-bloodpressure.json.html) result, but the display is an intentional joining of two, so they share indicators visually when one or the other has an indicator like having comments, or modified, but get their own individual treatment for interpretation (criticality).

Reply:
We should be able to use remove the unused props (performeddate, etc.) here without issues. Will give it a test.